server/routes/Auth.ts

```diff
@@ -28,7 +29,13 @@ authRouter.get('/oauth-callback', (req: Request, res: Response) => {
         .post(`https://github.com/login/oauth/access_token`, body, opts)
         .then((response: ResObj) => response.data)
         // eslint-disable-next-line no-console
-        .then((data: object) => console.log('this is data', data))
+        .then((data: object) => {
+            console.log('this is data', data);
```

Comment:
To clarify, we still aren't doing anything with the github info correct?

Reply:
Yes, nothing is happening with Github yet. I actually removed it from the webpage as an option until it gets hooked up.